jax/experimental/jax2tf/jax2tf.py

```diff
@@ -107,9 +107,20 @@ def _safe_convert_to_tensor(val, dtype=None) -> TfVal:
 tf_impl_with_avals: Dict[core.Primitive,
                          Callable[..., Any]] = {}
 
+# XLA is not linked in all environments; when converting a primitive, if the
```

Comment:
I would extend this comment to say that if XLA is disabled then we try harder to use only standard TF ops, if they are applicable to the concrete use case.